response_operations_ui/forms.py

```diff
@@ -66,3 +66,9 @@ class EditCollectionExerciseDetailsForm(FlaskForm):
 class ChangeGroupStatusForm(FlaskForm):
     event = StringField('event')
     submit = SubmitField('Confirm')
+
+
+class EditSurveyDetailsForm(FlaskForm):
+    long_name = StringField('long_name')
+    short_name = StringField('short_name')
```

Comment:
Need to confirm if short name is allowed to have a space in it, if not will require validation.